content/source/assets/javascripts/analytics.js

```diff
@@ -13,4 +13,25 @@ document.addEventListener('turbolinks:load', function() {
       product: 'terraform'
     }
   })
+
+  track('.notification', function(el) {
+    var params = {
+      name: 'Alert Banner',
+      variant: getVariantTextFromHtml(el.innerHTML)
+    }
+    // Create a stringified `label` prop of the whole event for GA tracking purposes
+    var label = JSON.stringify(params)
+    params.label = label
```

Comment:
Really being nit-picky here (I see that you just swiped my example code here, so perhaps blame me for this), but do we need to create a variable for this?

```js
params.label = JSON.stringify(params)
```

Reply:
agreed